README.md

```diff
@@ -124,6 +125,26 @@ docker-compose stop config-editor && docker-compose rm config-editor
 
 For full documentation about the config editor, see the docs for the product at [github.com/cdr/code-server][code-server].
 
+#### Migration from venv
+
+If you are currently running in a venv it is easy to migrate to a docker container by copying the existing venv data to a docker mapped persistent volume.
+
+e.g
+
+```
+cp -r /home/octoprint/.octoprint/. /docker/octoprint
+```
+
+Remember to stop and disable the service for your venv
+
+```
+systemctl stop octoprint.service
+systemctl disable octoprint.service
+```
+
+Then map your copied folder to the right location inside the container (by default this is /octoprint/octoprint/) and start the container.
+Remember to also map the serial \ usb device as a persistent name, not dynamic names
```

Comment:
I have an article coming up about cgroups, udev rules, and more.  For now, I recommend striking this line and we can add something back to it like remember to configure your video devices (see more here link)